DDDEastAnglia/Views/Account/_ExternalLoginsPartial.cshtml

```diff
@@ -0,0 +1,49 @@
+﻿@model IEnumerable<DDDEastAnglia.Models.LoginMethodViewModel>
+
+@if (Model.Any())
+{
+    foreach (var loginMethod in Model)
+    {
+        <div style="margin-bottom: 10px;">
+            <span style="line-height: 30px;">
+                <i class="@loginMethod.Icon"></i> @loginMethod.Name
+            </span>
+
+            <span style="position: absolute; left: 150px;">
+                @if (loginMethod.ProviderUserId == null)
+                {
+                    using (Html.BeginForm("ExternalLogin", "Account", new {ViewBag.ReturnUrl}))
+                    {
+                        @Html.AntiForgeryToken()
+                        <button type="submit" 
+                                class="btn btn-success" 
+                                name="provider"
+                                value="@loginMethod.ProviderName"
+                                title="Log in using @loginMethod.Name">
```

Comment:
That doesn't really make sense. By clicking the button, you are allowing yourself to log in with the provider you have clicked on (e.g. by clicking on the GitHub button, I am allowing myself to log into the DDDEA site using GitHub).